Source/Common/IGListDiff.h

```diff
@@ -30,6 +30,17 @@ typedef NS_ENUM(NSInteger, IGListDiffOption) {
 };
 
 /**
+ Comparison behavior options.
+ */
+typedef NS_OPTIONS(NSInteger, IGListDiffBehavior) {
+    IGListDiffBehaviorDefault = 0,
+    /**
+     Report moves which can be applied incrementally. O(n^2) complexity. By default, old and new positions for each object are reported.
```

Comment:
should be "**Reports** moves...."

> By default, old and new positions for each object are reported.

Can you elaborate on this more? I don't think it's clear what this option provides. 😊